pkg/manifest/installer.go

```diff
@@ -565,7 +566,7 @@ func isPodReady(pod *v1.Pod) bool {
 
 func deploymentsReady(deployments []deployment) bool {
 	for _, v := range deployments {
-		if !(v.replicaSets.Status.ReadyReplicas >= *v.deployment.Spec.Replicas-deploymentutil.MaxUnavailable(*v.deployment)) {
+		if !(v.replicaSets.Status.ReadyReplicas >= *v.deployment.Spec.Replicas) {
```

Comment:
nit: can you invert the conditional here so its easier to read? e.g.

```golang
if v.replicaSets.Status.ReadyReplicas < *v.deployment.Spec.Replicas {
}
```

Reply:
done